components/automate-chef-io/content/docs/backup.md

```diff
@@ -183,10 +187,10 @@ If backups are stored in S3:
 chef-automate backup restore s3://bucket_name/path/to/backups/BACKUP_ID
 ```
 
-To restore a backup of an [airgapped installation]({{< relref "airgapped-installation.md" >}}), you must specify the [Airgap Installation Bundle]({{< relref "airgapped-installation.md#create-an-airgap-installation-bundle" >}}) the installation is using:
+To restore a backup of an [airgapped installation]({{< relref "airgapped-installation.md" >}}), you must extract your single-file backup archive into the default backup directory (`/var/opt/chef-automate/backups`) and specify the [Airgap Installation Bundle]({{< relref "airgapped-installation.md#create-an-airgap-installation-bundle" >}}) the installation is using:
```

Comment:
I think this is fine. If we wanted to be even more explicit, we could rewrite this section as something like:

```
If you are extracting from a single-file backup archive, you must first extract your archive into the default backup directory before running the restore command: <command>. Otherwise, run the command <command>
```
It's repetitive, but I'm inclined to "go here and do this" directions because I believe no one reads operational docs except to find out exactly what they need to type. (@mjingle might have other opinions)